lib/elixir/test/elixir/io/ansi/docs_test.exs

```diff
@@ -550,6 +550,20 @@ defmodule IO.ANSI.DocsTest do
       assert format_markdown("a | b | c\nd | e") == "a | b | c\nd | e |  \n\e[0m"
     end
 
+    test "table with escaped \"|\" inside cell" do
+      table = "a | smth\\|smth_else | c\nd | e | f"
```

Comment:
Could please you format the table accordingly as it would be in the docs?

Reply:
The examples below use one-liners too, so this is fine. :)

Reply:
Done